src/universal/modules/landing/containers/Landing/LandingContainer.js

```diff
@@ -4,46 +4,94 @@ import {connect} from 'react-redux';
 import Helmet from 'react-helmet';
 import {head, auth0} from 'universal/utils/clientOptions';
 import {push} from 'react-router-redux';
-import {loginUserError, loginAndRedirect} from 'universal/modules/auth/ducks/auth';
+import {cashay} from 'cashay';
+import ActionHTTPTransport from 'client/ActionHTTPTransport';
+import {localStorageVars} from '../../../../utils/clientOptions';
 
-const mapStateToProps = state => ({
-  isAuthenticated: state.getIn(['auth', 'isAuthenticated']),
+const queryString = `
+query {
+  cachedUserAndToken: getUserWithAuthToken(authToken: $authToken) {
+    authToken,
+    user {
+      id,
+      cachedAt,
+      cacheExpiresAt,
+      createdAt,
+      updatedAt,
+      userId,
+      email,
+      emailVerified,
+      picture,
+      name,
+      nickname,
+      identities {
+        connection,
+        userId,
+        provider,
+        isSocial,
+      }
+      loginsCount,
+      blockedFor {
+        identifier,
+        id,
+      }
+    }
+  }
+}`;
+
+const mutationHandlers = {
+  updateUserWithAuthToken(optimisticVariables, dataFromServer, currentResponse) {
+    if (dataFromServer) {
+      currentResponse.cachedUserAndToken = dataFromServer.updateUserWithAuthToken;
+      return currentResponse;
+    }
+  }
+}
+
+const cashayOptions = {
+  component: 'AppContainer',
+  variables: {
+    authToken: response => response.cachedUserAndToken.authToken
+  },
+  mutationHandlers
+};
+
+const mapStateToProps = () => ({
+  response: cashay.query(queryString, cashayOptions)
 });
 
 @connect(mapStateToProps)
 export default class LandingContainer extends Component {
   static propTypes = {
     // children: PropTypes.element,
     isAuthenticated: PropTypes.bool.isRequired,
-    dispatch: PropTypes.func.isRequired
+    dispatch: PropTypes.func.isRequired,
+    userId: PropTypes.string
   };
 
   handleOnMeetingCreateClick = () => {
-    const {isAuthenticated, dispatch} = this.props;
-    if (isAuthenticated) {
-      dispatch(push('/signin/create_team_and_meeting'));
+    const {isAuthenticated, dispatch, userId} = this.props;
+    if (isAuthenticated && userId) {
```

Comment:
currently not working, since we'll need to stick an auto-login-from-localStorage-token in the `ActionContainer`